fix(media): Add alt text support for accessibility

klausi opened the pull request:
See https://blog.twitter.com/developer/en_us/a/2016/alt-text-support-for-twitter-cards-and-the-rest-api.html

Not sure why the Twitter docs don't mention the ext_alt_text field at https://developer.twitter.com/en/docs/tweets/data-dictionary/overview/entities-object#media , maybe they forgot to document it there?

I have not looked into test coverage yet.

QuietMisdreavus commented on src/entities.rs:
```diff
@@ -106,6 +106,9 @@ pub struct MediaEntity {
     ///video.
     pub video_info: Option<VideoInfo>,
+    ///Alt text for accessibility. If alt text isn't available for a given image, this will be
+    ///`None`.
```
Docs suggestion:

> Media alt text, if present.

I've seen alt text attached to videos, so it's probably best not to assume it'll only be attached to images.

klausi commented:
Thanks a lot! Twitter has never updated the blog post about alt text, the API parameter include_ext_alt_text is still required. See https://twittercommunity.com/t/is-include-ext-alt-text-still-required/74651

Added a sample tweet file, added a parse test case. Slapped the include_ext_alt_text on all the placeses that I think it is needed. Tested tweet show and user timeline manually with that, alt text is included.
